src/utils/normalizeScreenshot.js

```diff
@@ -18,16 +19,19 @@ async function normalizeRetinaScreenshot(browser, screenDimensions, base64Screen
     return base64Screenshot;
 }
 
-async function normalizeIOSScreenshot(browser, screenDimensions, base64Screenshot) {
-  const toolbarHeight = 44; // bottom toolbar has always a fixed height of 44px
-  const addressbarHeight = 44; // bottom toolbar has always a fixed height of 44px
+async function normalizeMobileScreenshot(browser, screenDimensions, base64Screenshot) {
+  const mobileFixedHeights = (browser.isAndroid) ? fixedHeights.android : fixedHeights.iOS;
+  const toolbarHeight = mobileFixedHeights.toolbarHeight;
```

Comment:
I need to update the test case screenshots.
I don't know why it appears in some cases either, but when I run locally 5.1 it doesn't appear. SauceLabs emulator for 5.1 android however, shows the following. Note the footer.
![android_footer](https://user-images.githubusercontent.com/496933/29093135-9b3b8f9c-7c3d-11e7-8bf2-2a5be3839245.png)